pkg/console/install_panels.go

```diff
@@ -2103,6 +2108,11 @@ func addInstallPanel(c *Console) error {
 				}
 			}
 
+			if !isDefaultRouteExist() {
+				printToPanel(c.Gui, fmt.Sprint("No default route found. Please check the router setting on the DHCP server"), installPanel)
```

Comment:
Same comment as above about not needing `fmt.Sprint`.

Also, I'd suggest adding `logrus.Error("No default route [...]")` so the same message goes to both the panel, and to `/var/log/console.log`